CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ find_package(ECM REQUIRED NO_MODULE)
 
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake)
 
+option(WITH_DEMO "Build the demo app" ON)
```

Comment:
This is a misnomer, the app/ directory does not contain a demo app but rather a bunch of factorized boilerplate for easy app creation. This gets used by pretty much all apps, here for example: https://github.com/AsteroidOS/asteroid-helloworld/blob/master/src/main.cpp#L22

I can see how disabling its build can help move along the watchface creation but ultimately one day we'll also want this directory to build for desktop so that apps can be built against it on a desktop computer too.

For now, you can name it `WITH_ASTEROIDAPP` for example, and maybe add a comment that explains this provides the AsteroidApp class.

Reply:
Ah, I didn't really understand what it was for.  I've renamed the CMake option and altered the text and the README to match.  I've also started on some QDoc documentation so I'll add some explanation of this class to it.  Thanks!